skrub/datasets/_ken_embeddings.py

```diff
@@ -99,6 +99,7 @@ def get_ken_types(
         search_result = unique_types.X[unique_types.X["Type"].str.contains(search)]
     if exclude is not None:
         search_result = search_result[~search_result["Type"].str.contains(exclude)]
+    search_result["Type"] = search_result["Type"].str.replace("<", "").str.replace(">", "")
```

Comment:
Wouldn't it be better to do ".str[1:-1]": what if there was a "<" inside the string (maybe it's impossible?)

Reply:
These are extracted from Wikipedia URL's, thus we have the "_" instead of spaces. 
And having ">" or "<" symbols is apparently not possible:
https://stackoverflow.com/questions/1547899/which-characters-make-a-url-invalid#1547940